packages/flutter/lib/src/rendering/object.dart

```diff
@@ -326,6 +325,29 @@ class PaintingContext extends ClipContext {
     _containerLayer.append(_currentLayer!);
   }
 
+  /// Adds a [CompositionCallback] for the current [ContainerLayer] used by this
+  /// context.
+  ///
+  /// Composition callbacks are called whenever the layer tree containing the
```

Comment:
nit: some of these docs seem duplicated from Layer.addCompositionCallback. Consider using a macro.

Reply:
Done